Src/ILGPU/Runtime/Cuda/CudaNativeMethods.cs

```diff
@@ -12,11 +12,14 @@
 #pragma warning disable IDE1006 // Naming Styles
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace ILGPU.Runtime.Cuda
 {
     #region Enums
 
-    enum StreamFlags
+    [SuppressMessage("Naming", "CA1717")]
```

Comment:
ISSUE: Should use the full supression `[SuppressMessage("Naming", "CA1717:Only FlagsAttribute enums should have plural names")]` or maybe add the `[Flags]` attribute instead?

Reply:
Didn't know that was an attribute. I'll get right to it.

Reply:
I can't seem to find the flags attribute. Was it recently added? This branch is behind quite a few commits as I forgot to pull before starting...